bindata/v4.1.0/kube-apiserver/apiserver.openshift.io_deprecatedapirequests.yaml

```diff
@@ -0,0 +1,247 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  name: deprecatedapirequests.apiserver.openshift.io
+spec:
+  group: apiserver.openshift.io
+  names:
+    kind: DeprecatedAPIRequest
+    listKind: DeprecatedAPIRequestList
+    plural: deprecatedapirequests
+    singular: deprecatedapirequest
+  scope: Cluster
+  versions:
+  - name: v1
+    schema:
+      openAPIV3Schema:
+        description: DeprecatedAPIRequest tracts requests made to a deprecated API.
+          The instance name should be of the form `resource.version.group`, matching
+          the deprecated resource.
+        type: object
+        required:
+        - spec
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: spec defines the characteristics of the resource.
+            type: object
+            properties:
+              removedRelease:
+                description: removedRelease is when the API will be removed.
+                type: string
+                maxLength: 64
+                minLength: 3
+                pattern: ^[0-9][0-9]*\.[0-9][0-9]*$
```

Comment:
this allows `99.99` right? so the max length here is 5 ?

Reply:
pretty sure the * allows as many as you like.

Reply:
At least one digit on each side.